src/pfe/file-watcher/server/src/controllers/projectsController.ts

```diff
@@ -994,17 +994,17 @@ export function deleteFolder(dir: string): Promise<void> {
                 if (err) {
                     return reject(err);
                 }
-                // Delete all files in the directory
-                Promise.all(files.map((file) => {
-                    return deleteFile(dir, file);
-                })).then(() => {
-                    rimraf(dir, (err) => {
-                        if (err) {
-                            return reject(err);
-                        }
-                        resolve();
-                    });
-                }).catch(reject);
+                // Delete log folder and log files inside log folder
```

Comment:
Why do you change the comment to log folder and log files only given that the dir has not been changed? Are we still deleting all files instead of just the logs?

Reply:
Yes, same behavior as before (delete both folder and files inside folder, not specific to log folder/file). I agree this comment is confusing as the the function is util function and should be shared by all callers, I will update the comment.